work/v1/types.go

```diff
@@ -79,9 +79,55 @@ type ManifestConfigOption struct {
 	ResourceIdentifier ResourceIdentifier `json:"resourceIdentifier"`
 
 	// FeedbackRules defines what resource status field should be returned.
+	// +optional
```

Comment:
This is technically breaking, but newer clients needed to handle the missing case anyway.  Please clarify if there is a difference between missing and empty (there shouldn't be a difference).